### ISSUE TITLE
Dependencies installing with npm... error: Request timed out after one minute

### ISSUE THREAD

**Issue by WintrCodes** [withastro/astro#6600]
### What version of `astro` are you using?

2.1.3

### Are you using an SSR adapter? If so, which one?

None

### What package manager are you using?

npm

### What operating system are you using?

Mac and Windows

### What browser are you using?

N/A

### Describe the Bug

When creating a new Astro project via 'npm create astro@latest', installing deps via the interactive CLI times out if you select 'yes' to install deps.

![image](https://user-images.githubusercontent.com/110974960/226331817-9292b659-421e-4056-92bf-c1a814fba6b7.png)

Workaround was to select 'no' and run 'npm i' after switching to the directory

Happened on Windows and MacOS Ventura

Requesting logs from Discord user (Peak)

Discord thread:
https://discordapp.com/channels/830184174198718474/1086276455773778020

### Link to Minimal Reproducible Example

Unable to replicate in StackBlitz

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by bluwy** [comment]
Looks like it's coming from:

https://github.com/withastro/astro/blob/1f749c942af4fcf138f9c4be07ce6b189c0a88fa/packages/create-astro/src/actions/dependencies.ts#L46

Maybe it's because the `npm install` actually took more than a minute to finish? Does that happen when you run `npm  i` separately?

**Comment by ematipico** [comment]
@WintrCodes, if you could share the contents of the npm logs, that would be amazing. It could help us understand what's the underlying issue.

**Comment by WintrCodes** [comment]
> Looks like it's coming from:
> 
> https://github.com/withastro/astro/blob/1f749c942af4fcf138f9c4be07ce6b189c0a88fa/packages/create-astro/src/actions/dependencies.ts#L46
> 
> Maybe it's because the `npm install` actually took more than a minute to finish? Does that happen when you run `npm i` separately?

When running `npm i` outside of the installer CLI, it installed with zero issue

**Comment by WintrCodes** [comment]
> I couldn't find the npm logs on my Mac; if someone knows where they'll be, I'll double check!

I couldn't find the npm logs on my Mac; if someone knows where they'll be, I'll double check!

**Comment by ematipico** [comment]
> > I couldn't find the npm logs on my Mac; if someone knows where they'll be, I'll double check!
> 
> I couldn't find the npm logs on my Mac; if someone knows where they'll be, I'll double check!

If you check the screenshot you pasted,  the error gives you a path to where npm/pnpm storer error logs.

**Comment by WintrCodes** [comment]
> > I couldn't find the npm logs on my Mac; if someone knows where they'll be, I'll double check!
> 
> If you check the screenshot you shared, the error gives you a path to where npm/pnpm storer error logs.

The screenshot is from another user in Discord and on a Windows device. I asked them (in the thread) to provide the log

**Comment by KickAgency** [comment]
I just ran into this issue on my first attempt of trying out Astro.

![image](https://user-images.githubusercontent.com/124220427/228409738-77475589-677b-4ec8-a5fb-3d6c6f9c14f7.png)

**Comment by anand007s** [comment]
I am also facing the same issue

**Comment by luiseduardoaugusto** [comment]
I found that i was running a dev server in another terminal. close that an try again

**Comment by akinors** [comment]
I had the same problem using with powershell, but after i switched to CMD it worked.

**Comment by prantikseal** [comment]
didnt work for me :(

**Comment by jooemrp** [comment]
![image](https://github.com/withastro/astro/assets/75409382/f68360e4-d667-4874-9288-ebecabf4a147)

I choose "no" when asked "install dependencies?" 
and manually doing `npm i` works for me.

**reify:typescript: timing reifyNode:node_modules/shiki Completed in 142832ms** <-- looks like this part is the reason why install dependencies timeout.

**Comment by technologyconsulting** [comment]
I'm now running into this using the latest version of Astro. Whilst running npm i works this isn't a fix as the docs don't mention it as the default / must do thing to get up and running.

I am trying Astro for the first time and problems on initial 'basic' setup is the last thing I wish to see.

**Comment by HarryAdney** [comment]
Yeah, I'm getting this too on Windows 10 (I know, don't judge me).

Tried again following advice here - say no to installing dependencies and installing seperately works fine.

**Comment by ingbioservicios** [comment]
"astro": "^5.7.0",

Yeah, I'm getting this too on Windows 11
